cli/main.c

```diff
@@ -731,38 +731,6 @@ static int bw(int argc, char **argv)
 	return 0;
 }
 
-#define CMD_DESC_LTMON "dump port ltssm state machine"
-static int ltmon(int argc, char **argv)
```

Comment:
Didn't we just add it in this PR? Reviewing code that gets removed or fixed later in the PR just wastes your reviewers time.